RealmSwift/Combine.swift

```diff
@@ -310,6 +310,35 @@ public func changesetPublisher<T: Object>(_ object: T) -> RealmPublishers.Object
     RealmPublishers.ObjectChangeset<T>(object)
 }
 
+extension Realm {
+    @available(OSX 10.15, watchOS 6.0, iOS 13.0, iOSApplicationExtension 13.0, OSXApplicationExtension 10.15, tvOS 13.0, *)
+    public func asyncWrite<S: Scheduler>(on scheduler: S = DispatchQueue.main as! S, _ block: @escaping (Realm) -> Void) -> RealmPublishers.AsyncWriteFromRealm<S> {
+        RealmPublishers.AsyncWriteFromRealm(scheduler, configuration: configuration, block)
+    }
+    @available(OSX 10.15, watchOS 6.0, iOS 13.0, iOSApplicationExtension 13.0, OSXApplicationExtension 10.15, tvOS 13.0, *)
+    public func asyncWrite<S: Scheduler, Confined: ThreadConfined>(
+        on scheduler: S = DispatchQueue.main as! S,
+        _ confined: Confined,
+        _ block: @escaping (Realm, Confined) -> Void) -> RealmPublishers.AsyncWriteRealmCombine<S, Confined> {
+        RealmPublishers.AsyncWriteRealmCombine(scheduler,
+                                               configuration: configuration,
+                                               confined: confined,
+                                               block)
+    }
+    @available(OSX 10.15, watchOS 6.0, iOS 13.0, iOSApplicationExtension 13.0, OSXApplicationExtension 10.15, tvOS 13.0, *)
+    public func asyncWrite<S: Scheduler, Confined: ThreadConfined, Confined2: ThreadConfined>(
```

Comment:
Does this mean you can't pass 3 objects to `asyncWrite`?

Reply:
Because of a lack of variadic generics support in Swift, we will have to write many overloads for this method that each take n amount of Objects/ThreadConfined types. In this prototype, I simply wrote two because it was easier and I was trying to see people's appetite for this API.

Reply:
Gotcha, makes sense - just wanted to understand if I'm misunderstanding the Swift syntax.